third_party/terraform/resources/resource_container_cluster.go.erb

```diff
@@ -1170,6 +1175,16 @@ func resourceContainerClusterCreate(d *schema.ResourceData, meta interface{}) er
 	timeoutInMinutes := int(d.Timeout(schema.TimeoutCreate).Minutes())
 	waitErr := containerOperationWait(config, op, project, location, "creating GKE cluster", timeoutInMinutes)
 	if waitErr != nil {
+		// Check if the create operation failed because Terraform was prematurely terminated. If it was we can persist the
+		// operation id to state so that a subsequent refresh of this resource will wait until the operation has terminated
+		// before attempting to Read the state of the cluster. This allows a graceful resumption of a Create that was killed
+		// by the upstream Terraform process exiting early such as a sigterm.
+		select {
+		case <-config.context.Done():
+			log.Printf("[DEBUG] Persisting %s so this operation can be resumed \n", op.Name)
+			d.Set("operation", op.Name)
```

Comment:
Is it normal for us to include newlines in these?

Reply:
There's a lot of log.Println used which already has the newline added, but since I'm using Printf I included it manually. Since I don't see any other Printf call using this but my terminal isn't 1 very long line I assume Terraform core is adding the newline in somewhere.